prdoc/pr_4151.prdoc

```diff
@@ -0,0 +1,10 @@
+title: "[pallet-broker] Use saturating math in input validation"
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Use saturating in the pallet-broker input validation of the `drop_history` extrinsic.
```

Comment:
How does this change the functionality of the extrinsic? Could we add that here?
Does something change for runtime users?